challenges/fizzBuzzTree/fizz-buzz-tree.js

```diff
@@ -0,0 +1,55 @@
+// Node
+class Node {
+  constructor(value, left = null, right = null) {
+    this.value = value
+    this.left = left
+    this.right = right
+  }
+}
+
+//Binary Tree class
+class BinaryTree {
+  constructor() {
+    this.root = null;
+  }
+
+  fizzBuzzTree() {
+    // make new tree
+    let newTree = new BinaryTree
+    newTree.root = this.root
+
+    // traverse tree
+    let traverse = (node) => {
+      if (node === null) {
+        return null
+      }
+
+      if (node.value%15 === 0) {
+        node.value = 'FizzBuzz'
+      } else if (node.value%3 === 0) {
+        node.value = 'Fizz'
+      } else if (node.value%5 === 0) {
+        node.value = 'Buzz'
+      } else {
+        node.value = node.value.toString()
```

Comment:
Why are you converting this to a string? Your test for keeping the original tree isn't passing because it is expecting to get a 1 but it is getting "1"

Reply:
The requirements wanted us to convert to string if the value wasn't %3 or %5 
Modified my code so the test will work see this pr
https://github.com/astrokd/data-structures-and-algorithms/pull/46